src/Config/DatabasePrimitives/DatabaseObject.cs

```diff
@@ -125,6 +125,7 @@ public class ParameterDefinition
 {
     public Type SystemType { get; set; } = null!;
     public DbType? DbType { get; set; }
```

Comment:
What purpose does DbType have now as opposed to SqlDbType?

Do we need a MsSqlspecific derived Parameter Definition?

Reply:
DbType here is the .Net DbType, while SqlDbType is SqlDb specific dbType. For example: .Net DbType doesn't differentiate between varchar and nvarchar, it uses just String type for both. So, here's how SqlDbType is helpful.

Reply:
So, if we have SqlDbType available for an MsSql parameter, do we need the DbType as well?

Reply:
Bloating a ParamDefinition/ColumnDefinition with fields not relevant for every database seems not to be the best approach here.

Reply:
we would have to check it's where it is being used. Could be taken as a seperate clean up PR.